app/visualization.py

```diff
@@ -0,0 +1,77 @@
+from pandas.core.frame import DataFrame
+from app.requester import ExchangeRateRequester
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+class Grapher:
+    def __init__(self) -> None:
+        self.exchange_rate_requester = ExchangeRateRequester()
+
+    def _get_historical_rates_df(self, transactions) -> DataFrame:
+        assert transactions, "There are no transactions"
+
+        first_transaction = transactions[0].date
+        historical_rates_date = self.exchange_rate_requester.get_historical_bids(first_transaction)
+        df = pd.DataFrame(list(historical_rates_date.items()), columns=["date", "rate"])
+
+        return df
+
+    def _get_operations_df(self, transactions) -> DataFrame:
+        assert transactions, "There are no transactions"
+
+        data_frame = self._get_historical_rates_df(transactions)
```

Comment:
`data_frame` -> `operations_df`

Reply:
dodaj kolumne buy_quota, sell_quota in EUR or one column with +/- sign

Reply:
done